apps/api/src/server/migration/migrators/folder-migrator.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @typedef {import('pins-data-model').Schemas.Folder} FolderModel
+ * @typedef {import('apps/api/src/database/schema.d.ts').Folder} Folder
+ */
+
+import { getCaseIdFromRef } from './utils.js';
+import * as folderRepository from '#repositories/folder.repository.js';
+import { defaultCaseFoldersForMigration } from './folder/folder.js';
+
+/**
+ * @param caseReference
```

Comment:
Looks like it should be
```suggestion
 * @param {string} caseReference
```